doc/admin-guide/files/records.config.en.rst

```diff
@@ -4152,7 +4152,40 @@ HTTP/2 Configuration
    :reloadable:
    :units: bytes
 
-   The initial window size for inbound connections.
+   The initial HTTP/2 stream window size for inbound connections that |TS| as a
+   receiver advertises to the peer. See IETF RFC 9113 section 5.2 for details
+   concerning HTTP/2 flow control. See
+   :ts:cv:`proxy.config.http2.flow_control.policy.in` for how HTTP/2 stream and
```

Comment:
I personally prefer a dot for this in/out thing, but we currently use an underscore. Let's consistently use an underscore.

Reply:
Makes sense. Done.